src/dini/defaults.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+/**
+ * Sets the defaults for W3C specs
```

Comment:
```suggestion
 * Sets the defaults for DINI specs
```

Reply:
I'll fix this locally.

Reply:
I'll fix this since I've got to make some more changes anyways.